phac-nml/irida
# Add fast5 sync

## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Added fast5 data to project synchronization.

**To test this PR**: 
1) Upload some fast5 data to a sample in a project
2) Synchronize it using the project  synchronization features. 
3) Ensure the fast5 data gets sync'd from the remote installation.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

Fixes #716 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:
